### ISSUE TITLE
The dist_mat value is approximately equal to 0.25

### ISSUE THREAD

**Issue by Adorablepet** [dapengchen123/crf_affinity#5]
Why I calculated dist_mat values near 0.25, whether or not the same person.But in the work, both local similarity and group similarity are assumed to be within the range(0, 1).The higher values the similarities are, the more likely the two images belong to a same person.

**Comment by dapengchen123** [comment]
@Adorablepet  which line the "dist_mat" is related to?

**Comment by Adorablepet** [comment]
@dapengchen123 evaluator/msevaluator.py/def compute_distmat

**Comment by dapengchen123** [comment]
@Adorablepet  These similarity scores are about the testing data, which are less discriminative compared with the training data. That is , the situation in testing is not always as ideal as that in training.  If the final results are normal, this should not be the problem. Another reason may be that we linearly combine the 3 local similarities, the final similarity score is also based the linear weights.

**Comment by Adorablepet** [comment]
@dapengchen123 According to these similar scores, it is completely different whether they are the same person. For example, different people score 0.25, the same person is 0.25. And the alphas I used when calculating 0.167, is not related to this. Because the value does not seem to be written to the model. I calculated it based on the source code.

**Comment by dapengchen123** [comment]
@Adorablepet   see https://github.com/dapengchen123/crf_affinity/blob/6081acc2a64092f16550467b7701170c732b3e8e/reid/evaluator/msevaluator.py#L240. The final score is related to alpha. Overall, I also want to know whether there are some problems with the final results?

**Comment by Adorablepet** [comment]
@dapengchen123 I have a question that is very doubtful. Since the final score is only related to alphas, why is the weights (including alphas and beta) trained by the crfmodel to be 'softmax' function when calculating alphas? If the ’softmax‘ function only uses alphas, the final score will be higher, the difference will be larger, personal advice.
